src/css/pages/loginpage.css

```diff
@@ -0,0 +1,6 @@
+.login-area {
+    padding: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5714005)